volume/mounts/parser_test.go

```diff
@@ -299,7 +299,6 @@ func TestParseMountRaw(t *testing.T) {
 	tester := func(parser Parser, set parseMountRawTestSet) {
 
 		for _, path := range set.valid {
-
```

Comment:
Looks like this is the only change in this file and not related; would it be possible to update the test to verify the new behaviour?